Makefile

```diff
@@ -298,15 +298,18 @@ endif
 # Usage:
 #	make bench-daily TO=/path/to/file.json
 bench-daily:
-	go test -tags intest github.com/pingcap/tidb/pkg/session -run TestBenchDaily -bench Ignore --outfile bench_daily.json
+	go test -tags intest github.com/pingcap/tidb/pkg/distsql -run TestBenchDaily -bench Ignore --outfile bench_daily.json
 	go test -tags intest github.com/pingcap/tidb/pkg/executor -run TestBenchDaily -bench Ignore --outfile bench_daily.json
 	go test -tags intest github.com/pingcap/tidb/pkg/executor/test/splittest -run TestBenchDaily -bench Ignore --outfile bench_daily.json
-	go test -tags intest github.com/pingcap/tidb/pkg/tablecodec -run TestBenchDaily -bench Ignore --outfile bench_daily.json
 	go test -tags intest github.com/pingcap/tidb/pkg/expression -run TestBenchDaily -bench Ignore --outfile bench_daily.json
+	go test -tags intest github.com/pingcap/tidb/pkg/planner/core/tests/partition -run '^[^T]' -bench . -benchmem \
+		| awk 'BEGIN{printf("[")} /^Benchmark.*ns\/op/ {if (found>0) {printf(",")}; sub(/-[0-9]*/,"",$$1); printf("{\"Name\":\"%s\",\"NsPerOps\":%d,\"AllocsPerOp\":%d,\"BytesPerOp\":%d}",$$1,$$3,$$7,$$5); found++ } END{printf("]")}' \
+		> pkg/planner/core/tests/partition/bench_daily.json
```

Comment:
I could not find a way to run SubBenchmarks to be reported as separate tests, so I am just running it as command line bench marks and transforming the output to json through old fashioned texts processing...

Reply:
Maybe we should remove the awk script with https://pkg.go.dev/go.bobheadxi.dev/gobenchdata#readme-command-line-interface ?